### ISSUE TITLE
File upload does not work correctly

### ISSUE THREAD

**Issue by hakonhagland** [mschilli/net-google-drive-simple#47]
I am trying to upload a simple text file:
```
my $gd = Net::Google::Drive::Simple->new( oauth => $oauth );
my $file = 'hello.txt';
my $parent = 'root';
my $file_id = $gd->file_upload( $file, $parent ) or die "Upload failed: $!";
```
The upload does not report any errors, but when I check the uploaded file from the Google Drive web interface in my browser, I can see that the content of the file is either missing or contains `Content` of `Content-Type` which is not what was in the file.

**Comment by hakonhagland** [comment]
Added a possible fix, see PR #48.

**Comment by atoomic** [comment]
thanks this is now fixed
